llama_stack/distribution/providers.py

```diff
@@ -51,18 +51,22 @@ async def list_providers(self) -> ListProvidersResponse:
                 # Skip providers that are not enabled
                 if p.provider_id is None:
                     continue
+                # Filter out "metrics" to be shown in config duplicated
+                metrics_url = p.config.get("metrics")
```

Comment:
What if `metrics` is not configured? Should we display something else than null?